index.js

```diff
@@ -1012,9 +1043,45 @@ class GithubScm extends Scm {
                 username: hoek.reach(webhookPayload, 'sender.login'),
                 lastCommitMessage: hoek.reach(webhookPayload, 'head_commit.message') || '',
                 hookId,
-                scmContext: scmContexts[0]
+                scmContext: scmContexts[0],
+                ref: hoek.reach(webhookPayload, 'ref')
             };
+        case 'release': {
+            return {
+                action: 'release',
+                branch: hoek.reach(webhookPayload, 'release.target_commitish'),
+                checkoutUrl,
+                type: 'repo',
+                username: hoek.reach(webhookPayload, 'sender.login'),
+                hookId,
+                scmContext: scmContexts[0],
+                ref: hoek.reach(webhookPayload, 'release.tag_name')
+            };
+        }
+        case 'create': {
+            const refType = hoek.reach(webhookPayload, 'ref_type');
+
+            if (refType !== 'tag') {
+                winston.info('A %s event of %s is not parsable.', type, refType);
```

Comment:
How about `A %s event of %s is not available yet in Screwdriver.cd.` ?

Reply:
It looks good suggestion. I think `in Screwdriver.cd.`  has no meaning so I'll fix with `in scm-github plugin`.

Reply:
👍